Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/UpdateAcpiGnvs.c

```diff
@@ -896,15 +889,11 @@ PlatformUpdateAcpiGnvs (
   PlatformNvs->PpmFlags                         = CpuNvs->PpmFlags;
   SocUpdateAcpiGnvs ((VOID *)GnvsIn);
 
-  if (PchSciSupported) {
-    PlatformNvs->Rtd3Support                    = 0;
-    PlatformNvs->LowPowerS0Idle                 = 0;
-  }
-
+  PlatformNvs->Rtd3Support = PchSciSupported? 0 : 1;
   // If TCC is enabled, use the TCC policy from subregion
-  if (mTccDsoTuning) {
+  if (mTccDsoTuning)
     PlatformNvs->Rtd3Support     = mTccRtd3Support;
-    PlatformNvs->LowPowerS0Idle  = mTccLowPowerS0Idle;
-  }
 
+  PlatformNvs->LowPowerS0Idle = (UINT8) FeaturesCfgData->Features.S0ix;

```

Comment:
It looks there are several one will impact S0ix enable/disable
 mTccDsoTuning and DsoPolicy->Sstate; FeaturesCfgData->Features.S0ix; and the SKU.
Prefer adding a S0ix flag in PLATFORM_DATA and update this flag based on settings. so the consumer could directly use S0ix flag from  PLATFORM_DATA.

Reply:
Reworked. Added UpdateS0ixStatus();